css/_vertical_filmstrip_overrides.scss

```diff
@@ -50,9 +50,20 @@
             }
         }
 
+        /**
+         * Re-styles the local Video and invite button to better fit the
+         * vertical filmstrip layout.
+         */
         #filmstripLocalVideo {
+            bottom: 5px;
+            flex-direction: column-reverse;
             height: auto;
-            justify-content: flex-end;
+            justify-content: flex-start;
```

Comment:
You'll probably have to adjust the local stats and maybe remote stats dialogs. On the plus, it might end up looking better now.

Reply:
I think the remote video menu needs adjusting now too... Because remote videos start on the bottom in vertical filmstrip, that menu can get cut off if there are many options, which I can see happening with mute, kick, local volume, and remote control.